api/tests/routes/backoffice/pro_test.py

```diff
@@ -398,10 +399,24 @@ def test_can_search_offerer_by_name(self, authenticated_client):
         assert_offerer_equals(cards_text[0], self.offerers[2])  # Théâtre du Centre (most relevant)
         assert_offerer_equals(cards_text[1], self.offerers[11])  # Théâtre du Centaure (very close to the first one)
 
+    @override_features(WIP_EXTENDED_SEARCH=False)
+    def test_can_search_offerer_by_name_not_extended_search(self, authenticated_client):
+        self._create_offerers()
+
+        with assert_num_queries(self.expected_num_queries + 1):  # feature flag
+            response = authenticated_client.get(
+                url_for(self.endpoint, q="du Centre", pro_type=TypeOptions.OFFERER.name)
+            )
+            assert response.status_code == 200
+
+        cards_text = html_parser.extract_cards_text(response.data)
+        assert len(cards_text) >= 3  # Results may contain all "du centre", "Centre"
+        assert_offerer_equals(cards_text[0], self.offerers[2])  # Théâtre du Centre (most relevant)
+
     def test_can_search_offerer_by_name_and_department(self, authenticated_client):
         self._create_offerers()
 
-        with assert_num_queries(self.expected_num_queries):
+        with assert_num_queries(self.expected_num_queries + 1):  # feature flag
```

Comment:
C'est sur de nombreux tests, donc j'aurais préféré définir une constante plus haut : 
`expected_num_queries_with_ff = expected_num_queries + 1`
et ainsi l'utiliser clairement :
```suggestion
        with assert_num_queries(self.expected_num_queries_with_ff):
```